chef/cookbooks/hyperv/recipes/register_services_ceilometer.rb

```diff
@@ -4,15 +4,15 @@
   code <<-EOH
     if (-not (Get-Service "#{node[:service][:ceilometer][:name]}" -ErrorAction SilentlyContinue))
     {
-      New-Service -name "#{node[:service][:ceilometer][:name]}" -binaryPathName "`"#{node[:openstack][:bin]}\\#{node[:service][:file]}`" ceilometer-compute-agent `"#{node[:openstack][:ceilometer][:installed]}`" --config-file `"#{node[:openstack][:config]}\\ceilometer.conf`"" -displayName "#{node[:service][:ceilometer][:displayname]}" -description "#{node[:service][:ceilometer][:description]}" -startupType Automatic
+      New-Service -name "#{node[:service][:ceilometer][:name]}" -binaryPathName "`"#{node[:openstack][:bin]}\\#{node[:service][:file]}`" ceilometer-agent-compute `"#{node[:openstack][:ceilometer][:binary]}`" --config-file `"#{node[:openstack][:ceilometer][:config]}\\ceilometer.conf`"" -displayName "#{node[:service][:ceilometer][:displayname]}" -description "#{node[:service][:ceilometer][:description]}" -startupType Automatic
       Start-Service "#{node[:service][:ceilometer][:name]}"
       Set-Service -Name "#{node[:service][:ceilometer][:name]}" -StartupType Automatic
     }
   EOH
 end
 
-service "ceilometer-compute" do
+service "ceilometer-agent-compute" do
   service_name node[:service][:ceilometer][:name]
   action [:enable, :start]
-  subscribes :restart, "template[#{node[:openstack][:config].gsub(/\\/, "/")}/ceilometer.conf]"
+  subscribes :restart, "template[#{node[:openstack][:ceilometer][:config].gsub(/\\/, "/")}/ceilometer.conf]"
```

Comment:
Metrics/LineLength: Line is too long. [108/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Performance/StringReplacement: Use `tr` instead of `gsub`. (https://github.com/JuanitoFatas/fast-ruby#stringgsub-vs-stringtr-code)